third_party/ibis/ibis_addon/operations.py

```diff
@@ -185,14 +185,16 @@ def sa_format_hashbytes(translator, expr):
 BigQueryExprTranslator._registry[BitXor] = bq_reduction("BIT_XOR")
 BigQueryExprTranslator._registry[Hash] = format_hash_bigquery
 BigQueryExprTranslator._registry[HashBytes] = format_hashbytes_bigquery
+BigQueryExprTranslator._registry[RawSQL] = format_raw_sql
 AlchemyExprTranslator._registry[RawSQL] = format_raw_sql
 AlchemyExprTranslator._registry[HashBytes] = format_hashbytes_alchemy
 MSSQLExprTranslator._registry[HashBytes] = sa_format_hashbytes
+MSSQLExprTranslator._registry[RawSQL] = format_raw_sql
 BaseExprTranslator._registry[RawSQL] = format_raw_sql
 BaseExprTranslator._registry[HashBytes] = format_hashbytes_base
-BigQueryExprTranslator._registry[RawSQL] = format_raw_sql
 ImpalaExprTranslator._registry[RawSQL] = format_raw_sql
 ImpalaExprTranslator._registry[HashBytes] = format_hashbytes_hive
 OracleExprTranslator._registry[RawSQL] = sa_format_raw_sql
 TeradataExprTranslator._registry[RawSQL] = format_raw_sql
 TeradataExprTranslator._registry[HashBytes] = format_hashbytes_teradata
+PostgreSQLExprTranslator._registry[RawSQL] = format_raw_sql
```

Comment:
```suggestion
PostgreSQLExprTranslator._registry[RawSQL] = sa_format_raw_sql
```

Postgres also uses SQL Alchemy